modules/notify/main/index.scss

```diff
@@ -72,9 +72,8 @@ body.hx-modal-open > {
     cursor: pointer;
   }
 
-  .hx-notification-text {
+  .hx-notification-content {
     word-wrap: break-word;
-    margin: 0;
```

Comment:
I'm pretty sure this is hanging around from when it was using the `Layout` module for the alignment